From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Thu, 23 Jan 2020 16:43:13 -0500
Subject: tests/json: Bravo key does not have secret key material

* tests/json/t-keylist-secret.out.json: indicate correctly that the
"Bravo" test key does not have secret key material.

--

Some older versions of GnuPG are buggy, and will claim "secret: true"
for this key, even though the keyring in question has no secret key
material for it.

The underlying call to gpg looks something like:

    gpg --with-colons --with-secret --list-keys -- \
        alpha@example.net bravo@example.net

You can see the change in field 15 of the "pub" and "sub" output.

Signed-off-by: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
---
 tests/json/t-keylist-secret.out.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/json/t-keylist-secret.out.json b/tests/json/t-keylist-secret.out.json
index 5da20db..109eae3 100644
--- a/tests/json/t-keylist-secret.out.json
+++ b/tests/json/t-keylist-secret.out.json
@@ -91,7 +91,7 @@
             "can_sign":     true,
             "can_certify":  true,
             "can_authenticate":     true,
-            "secret":       true,
+            "secret":       false,
             "is_qualified": false,
             "protocol":     "OpenPGP",
             "fingerprint":  "D695676BDCEDCC2CDD6152BCFE180B1DA9E3B0B2",
